pennylane/devices/default_qutrit.py

```diff
@@ -0,0 +1,261 @@
+        # temporarily overwrite this method to bypass
+        # wire map that produces Wires objects
+        try:
+            mapped_wires = [self.wire_map[w] for w in wires]
+        except KeyError as e:
+            raise WireError(
+                f"Did not find some of the wires {wires.labels} on device with wires {self.wires.labels}."
+            ) from e
+
+        return mapped_wires
+
+    def define_wire_map(self, wires):
+        # temporarily overwrite this method to bypass
+        # wire map that produces Wires objects
+        consecutive_wires = range(self.num_wires)
+        wire_map = zip(wires, consecutive_wires)
+        return dict(wire_map)
+
+    def apply(self, operations, rotations=None, **kwargs):  # pylint: disable=arguments-differ
+        rotations = rotations or []
+
+        # apply the circuit operations
+        # Operations are enumerated so that the order of operations can eventually be used
+        # for correctly applying basis state / state vector / snapshot operations which will
+        # be added later.
+        for i, operation in enumerate(operations):  # pylint: disable=unused-variable
+            self._state = self._apply_operation(self._state, operation)
+
+        # store the pre-rotated state
+        self._pre_rotated_state = self._state
+
+        # apply the circuit rotations
+        for operation in rotations:
+            self._state = self._apply_operation(self._state, operation)
+
+    def _apply_operation(self, state, operation):
+        """Applies operations to the input state.
+
+        Args:
+            state (array[complex]): input state
+            operation (~.Operation): operation to apply on the device
+
+        Returns:
+            array[complex]: output state
+        """
+        if operation.base_name == "Identity":
+            return state
+        wires = operation.wires
+
+        if operation.base_name in self._apply_ops:
+            axes = self.wires.indices(wires)
+            return self._apply_ops[operation.base_name](state, axes, inverse=operation.inverse)
+
+        matrix = self._asarray(self._get_unitary_matrix(operation), dtype=self.C_DTYPE)
+
+        return self._apply_unitary(state, matrix, wires)
+
+    def _get_unitary_matrix(self, unitary):  # pylint: disable=no-self-use
+        """Return the matrix representing a unitary operation.
+
+        Args:
+            unitary (~.Operation): a PennyLane unitary operation
+
+        Returns:
+            array[complex]: Returns a 2D matrix representation of
+            the unitary in the computational basis.
+        """
+        return unitary.matrix()
+
+    @classmethod
+    def capabilities(cls):
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            model="qutrit",
+            supports_reversible_diff=True,
+            supports_inverse_operations=True,
+            supports_analytic_computation=True,
+            returns_state=True,
```

Comment:
Are there plans to support other interfaces, torch, tf, jax, ?

Reply:
Currently, I don't have it planned. However, I think that it is definitely possible, and will definitely become more relevant once parametrized operations are available for qutrit devices (spoiler alert: those are already implemented in another set of draft PRs, see [#2845](https://github.com/PennyLaneAI/pennylane/pull/2845), [#2846](https://github.com/PennyLaneAI/pennylane/pull/2846), [#2847](https://github.com/PennyLaneAI/pennylane/pull/2847)). If it is possible for `DefaultQubit`, I don't see why it won't be possible for `DefaultQutrit`.

Reply:
After several instances of unexpected qubit assumptions, I wouldn't be surprised if the interface logic does rely on it, and I would imagine that in some cases it might not be unlikely. Binary logic can generally compute significantly faster than general decimal logic, and it's possible that one or more of the interfaces rely on this for optimization. Just a guess at this point, but regardless, interfaces aren't high priority at the moment.